src/main/java/marquez/common/models/UrnPattern.java

```diff
@@ -48,7 +48,7 @@ void throwIfNoMatch(@NonNull String value) {
       throw new IllegalArgumentException(
           String.format(
               "urn (%s) must contain only letters (a-z, A-Z), numbers (0-9), periods (.), "
-                  + "or underscores (_) and be sperated by colons (:) with each part "
+                  + " underscores (_) or dashes (-) and be sperated by colons (:) with each part "
```

Comment:
_Minor_: Great to see that you updated the description as well! You'll want to remove the extra spacing:

```
" underscores (_) or dashes (-) ...",  should be "underscores (_) or dashes (-) ..."
```

Reply:
done